actions/image_partition_action.go

```diff
@@ -420,6 +437,15 @@ func (i ImagePartitionAction) Run(context *debos.DebosContext) error {
 	if err != nil {
 		return err
 	}
+
+	if i.PTUUID != "" {
+		command := []string{"sfdisk", "--disk-id", context.Image, i.PTUUID}
```

Comment:
do we need to install sfdisk in Dockerfile ?

Reply:
No, we don't need to do this.
`fdisk ` package contains `sfdisk` binary.
Please refer to https://packages.debian.org/bullseye/amd64/fdisk/filelist